src/config/2021.json

```diff
@@ -297,9 +316,17 @@
       "github": "christianliebel",
       "twitter": "christianliebel"
     },
+    "digitala11ies": {
+      "name": "digitala11ies",
```

Comment:
@digitala11ies would you like to be credited by your full name in the chapter, or any other personal info shared (website, twitter, etc)?